src/base/bittorrent/session.h

```diff
@@ -291,8 +291,8 @@ namespace BitTorrent
         bool isBandwidthSchedulerEnabled() const;
         void setBandwidthSchedulerEnabled(bool enabled);
 
-        uint saveResumeDataInterval() const;
-        void setSaveResumeDataInterval(uint value);
+        int saveResumeDataInterval() const;
```

Comment:
You should also remove the `static_cast` in appcontroller.cpp line 266.

Reply:
@Chocobo1 fixed. Also took the opportunity to remove another similar static cast elsewhere and convert the refresh interval-related settings to use `int`s instead of `uint`s.